post_processing/compare_outputs.jl

```diff
@@ -0,0 +1,91 @@
+import ArgParse
+using Test
+import ClimaAtmos as CA
+import ClimaComms
+import ClimaCore.InputOutput as InputOutput
+import ClimaCore.Fields as Fields
+function parse_commandline()
+    s = ArgParse.ArgParseSettings()
+    ArgParse.@add_arg_table s begin
+        "--output_folder_1"
+        help = "Output file for job 1"
+        arg_type = String
+        "--output_folder_2"
+        help = "Output file for job 2"
+        arg_type = String
+        "--t_end"
+        help = "Simulation end time. This must match the target jobs"
+        arg_type = String
+        "--state_working"
+        help = "A bool indicating that the state test is working [`true` [default], `false`]"
+        arg_type = Bool
+        default = true
+        "--diagnostics_working"
```

Comment:
I'm a bit confusing on how a user would use this option. Do they need to know if the diagnostics is working before hand? Is this just for us? Sorry, it's not that clear from the help string.

Reply:
Yeah, we should probably improve the name. It basically indicates that the diagnostics are successfully compared, meaning it's tested and there's zero difference between the two outputs. Maybe `test_diagnostics` or `compare_diagnostics`?

Reply:
Oh, yes, I like `test_diagnostics` or `compare_diagnostics` much better!